### ISSUE TITLE
This repo should be archived?

### ISSUE THREAD

**Issue by jcrist** [conda-forge/ibis-bigquery-feedstock#12]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

The `ibis-bigquery` package is now managed as part of the `ibis-framework-feedstock`, this repo can probably be archived.

### Installed packages

```shell
N/A
```

### Environment info

```shell
N/A
```

**Comment by tswast** [comment]
Yes, as a maintainer of both the upstream and this feedstock, I believe it should. CC @conda-forge/core
